test/e2e/providers/azure/azure.go

```diff
@@ -124,3 +125,16 @@ func (p *Provider) StorageSupport() bool {
 func (p *Provider) CreatePVC(_ client.Interface, _ string) (*core.PersistentVolumeClaim, error) {
 	return nil, fmt.Errorf("storage not supported on azure")
 }
+
+// getImageSKU returns the SKU based on the Windows Server version
+func getImageSKU(windowsServerVersion windows.ServerVersion) string {
+	switch windowsServerVersion {
+	case windows.Server2019:
+		// 2019 images without the containers feature pre-installed cannot be used due to
+		// https://issues.redhat.com/browse/OCPBUGS-13244
+		return "2019-datacenter-with-containers-smalldisk"
```

Comment:
The `with-containers` images are getting removed any time now. Azure jobs testing with 2019 would be blocked. Should we explore alternatives before introducing this i.e. moving 2019 testing away from azure?

Reply:
I am aware of that but there is no easy fix that I can add, which is why I created [OCPBUGS-13244](https://issues.redhat.com/browse/OCPBUGS-13244), otherwise this is going to hold up the AWS 2022 support. Worst case if tests start failing, we can flip back to 2022 for Azure in the release repo.